javascript-practice/src/javascripts/models/product.js

```diff
@@ -0,0 +1,17 @@
+class ProductModel {
```

Comment:
This is product model, but I can not see anything relate to the properties of product?